frontend/packages/kubevirt-plugin/src/components/form/helper/container-source-help.tsx

```diff
@@ -4,8 +4,12 @@ import { isUpstream } from '@console/internal/components/utils';
 import {
   CENTOS7,
   CENTOS8,
```

Comment:
should we remove `CENTOS8` here as it's not in the template list anymore?

Reply:
we want to be backworkd compatible 2 versions back